Unexpected failure on USB0_DOWN test

```
_____________________________________________________________________________________________ test_usb0_down _____________________________________________________________________________________________

usb0_data = NetworkInterface(ifname='usb0', operstate='UP', mac='0e:bf:4d:9b:c3:65', mtu=1500, duplex=None, speed=None, ipv4_addr='169.254.42.1', ipv4_prefix=16)

    def test_usb0_down(usb0_data):
        """
        Test command:
            ip address
    
        Results:
            True - usb0 state is "DOWN"
            False - any other output
        """
>       assert usb0_data.operstate == "DOWN"
E       AssertionError: assert 'UP' == 'DOWN'
E         - DOWN
E         + UP

/opt/wlanpi-hwtest/lib/python3.9/site-packages/hwtest/automated/ethernet_test.py:92: AssertionError
```

```
wlanpi@wlanpi:~ $ ip link show usb0
3: usb0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP mode DEFAULT group default qlen 1000
    link/ether 0e:bf:4d:9b:c3:65 brd ff:ff:ff:ff:ff:ff
```

--- a/hwtest/automated/ethernet_test.py
+++ b/hwtest/automated/ethernet_test.py
@@ -47,11 +47,6 @@ def eth0_data():
     return get_ip_data("eth0")
 
 
-@pytest.fixture()
-def usb0_data():
-    return get_ip_data("usb0")
-
-
 def test_eth0_up(eth0_data):
     """
     Test commands:
@@ -78,27 +73,3 @@ def test_eth0_speed(eth0_data):
         False - any other output
     """
     assert eth0_data.speed == 1000
-
-
-def test_usb0_down(usb0_data):
-    """
-    Test command:
-        ip address
-
-    Results:
-        True - usb0 state is "DOWN"
-        False - any other output
-    """
-    assert usb0_data.operstate == "DOWN"
-
-
-def test_usb0_ip(usb0_data):
-    """
-    Test command:
-        ip address
-
-    Results:
-        True - IP is "169.254.42.1"
-        False - any other output
-    """
-    assert usb0_data.ipv4_addr == "169.254.42.1"

